hugegraph-api/src/main/java/com/baidu/hugegraph/api/job/ComputerDisAPI.java

```diff
@@ -76,10 +78,10 @@
     public Map<String, Object> post(@Context GraphManager manager,
                                     @PathParam("graph") String graph,
                                     JsonTask jsonTask) {
-        LOG.debug("Schedule computer dis job: {}", jsonTask);
-        E.checkArgument(K8sDriverProxy.isK8sApiEnabled() == true,
-                        "The k8s api is not enable.");
         checkCreatingBody(jsonTask);
+        E.checkArgument(K8sDriverProxy.isK8sApiEnabled(),
+                        "The k8s api is not enable.");
+        LOG.info("Schedule computer dis job: {}, graph is {}", jsonTask, graph);
```

Comment:
keep debug？

Reply:
mavbe save computer job log to server.log is better? (only include new job & delete job now)

set to debug is also OK